lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/sherllin/Dogtest.java

```diff
@@ -0,0 +1,100 @@
+package com.codedifferently.lesson7.sherllin;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class Dogtest {
+  private Dog dog;
+
+  @BeforeEach
+  void setUp() {
+    dog = new Dog(3, 10.5, "Brown", true, Breed.Poodle);
+  }
+
+  @Test
+  void testInvalidBreedException() {
+    assertThrows(
+        Dog.InvalidBreedException.class,
+        () -> {
+          throw new Dog.InvalidBreedException("Invalid breed");
+        });
+  }
+
+  @Test
+  void testDogDetails() {
```

Comment:
This test is checking too many things. In the future, use the `Arrange, Act, Assert` convention to ensure that you're only testing one thing.